packages/server/src/routes/sessions.js

```diff
@@ -16,12 +16,20 @@ const { isUSDRSuperAdmin } = require('../lib/access-helpers');
 
 // NOTE(mbroussard): previously we allowed 2 uses to accommodate automated email systems that prefetch
 // links. Now, we send login links through a clientside redirect instead so this should not be necessary.
-const MAX_ACCESS_TOKEN_USES = 1;
+// This has been updated to accommodate Microsoft Safe Links which will prefetch the link twice.
+const MAX_ACCESS_TOKEN_USES = 4;
 
 // the validation URL is sent in the authentication email:
 //     http://localhost:8080/api/sessions/?passcode=97fa7091-77ae-4905-b62e-97a7b4699abd
 //
 router.get('/', async (req, res) => {
+    const userAgent = req.headers['user-agent'] || '';
```

Comment:
I would recommend moving this out to a different function. It would be easy enough to test, and then we could add to it if we see this same issue with another service.